mod-password-validator/src/test/resources/domain/passwordvalidator/features/validate.feature

```diff
@@ -18,6 +18,20 @@ Feature: Test POST password validate
     Then status 200
     And match response.result == "valid"
 
+  Scenario: Should return 404 if user with given id not found
+    Given path 'password/validate'
+    And request password
+    And set password.userId = "WrongId"
+    When method POST
+    Then status 404
+
+  Scenario: Should return 422 if required fields not provided
+    Given path 'password/validate'
+    And request {}
+    When method POST
+    Then status 422
+    And match response.total_records == 2
```

Comment:
Please add more match related to errors. It looks like response contains two records